notebooks/nzh-20230521-00-ensembling.ipynb

```diff
@@ -0,0 +1,1004 @@
+{
```

Comment:
<p>Theres a way to do this in numpy as a vectorized operation </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dsgt-birdclef/birdclef-2023/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>